### PR TITLE
Example Workflow for scriptTask in ruby

### PR DESCRIPTION
**Pull request by rhundal7**
- Workflow Tested on Dev

### CODE REVIEW

File: examples/example-scripttask-ruby/nodes/end.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "a25c2fd3-cbbc-4604-b2aa-8f2565a6c76f",
+  "name": "End",
+  "description": "End of Ruby Test Workflow",
```
Comment (kaladay) [inline examples/example-scripttask-ruby/nodes/end.json:4]:
```suggestion
  "description": "End of Ruby Test Workflow.",
```

File: examples/example-scripttask-ruby/nodes/rb/runScript.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+print "Hello World from Ruby workflow!!"
```
Comment (kaladay) [inline examples/example-scripttask-ruby/nodes/rb/runScript.rb:1]:
```suggestion
print "\nHello World from Ruby workflow!!\n"
```

I think We should add new lines to see this more easily in the logs.

File: examples/example-scripttask-ruby/workflow.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "id": "6780cd32-a445-4fe0-990f-dcc9913ce99e",
+  "name": "Quick Ruby ScriptTest",
+  "description": "Test Ruby workflow",
```
Comment (kaladay) [inline examples/example-scripttask-ruby/workflow.json:4]:
```suggestion
  "description": "Test Ruby workflow.",
```

**Review by kaladay — CHANGES_REQUESTED** [review]

**Review by wwelling — APPROVED** [review]

**Review by qtamu — APPROVED** [review]

**Review by kaladay — APPROVED** [review]

**Review by qtamu — APPROVED** [review]

File: examples/example-scripttask-ruby/workflow.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "id": "6780cd32-a445-4fe0-990f-dcc9913ce99e",
+  "name": "Quick Ruby ScriptTest",
```
Comment (kaladay) [inline examples/example-scripttask-ruby/workflow.json:3]:
```suggestion
  "name": "Example Ruby ScriptTask",
```

File: examples/example-scripttask-ruby/nodes/end.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "a25c2fd3-cbbc-4604-b2aa-8f2565a6c76f",
+  "name": "End",
+  "description": "End of Ruby Test Workflow.",
```
Comment (kaladay) [inline examples/example-scripttask-ruby/nodes/end.json:4]:
```suggestion
  "description": "End of Example Ruby ScriptTask Workflow.",
```

File: examples/example-scripttask-ruby/nodes/start.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "1940f6af-b7db-41fa-855b-2b56c48e9561",
+  "name": "Start",
+  "description": "Start of Quick Ruby Test Workflow.",
```
Comment (kaladay) [inline examples/example-scripttask-ruby/nodes/start.json:4]:
```suggestion
  "description": "Start of Example Ruby ScriptTask Workflow.",
```

File: examples/example-scripttask-ruby/triggers/startTrigger.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
+    "id": "f72268ba-83ae-4253-9a4d-791a4bb107c7",
+    "name": "Quick Ruby Test Start Trigger.",
```
Comment (kaladay) [inline examples/example-scripttask-ruby/triggers/startTrigger.json:3]:
```suggestion
    "name": "Example Ruby ScriptTask Start Trigger.",
```

File: examples/example-scripttask-ruby/triggers/startTrigger.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+    "description": "REST Post Endpoint",
```
Comment (kaladay) [inline examples/example-scripttask-ruby/triggers/startTrigger.json:4]:
```suggestion
    "description": "A REST Post end point for the Example Ruby ScriptTask.",
```

File: examples/example-scripttask-ruby/workflow.json [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  "description": "Test Ruby workflow.",
```
Comment (kaladay) [inline examples/example-scripttask-ruby/workflow.json:4]:
```suggestion
  "description": "An example ScriptTask Workflow using Ruby.",
```

**Review by kaladay — CHANGES_REQUESTED** [review]

**Pull request merged**